test/crypto/sign.js

```diff
@@ -211,39 +228,31 @@ ${defaultSignature}
 		});
 
 		describe('#verify', () => {
-			const transactionForVerifyTwoSignatures = {
-				type: 0,
-				amount: '10',
-				fee: 10000000,
-				recipientId: '13356260975429434553L',
-				senderPublicKey: '215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca',
-				senderSecondPublicKey: '922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa',
-				timestamp: 39541109,
-				asset: {},
-				signature: 'e7027dbe9bb8ebcc1738c560fe0a09161d781d9bfc5df4e9b4ccba2d7a1febcd25ba663938c8d22d4902d37435be149cfb0fd69e7a59daf53469abe8f6509e0c',
-				signSignature: 'e88b4bd56a80de3b15220bdf0d1df0aa024a7a127ef07b8dc36a4e12d50e8eb338bc61ebe510ab15839e23f073cffda2a8c8b3d1fc1f0db5eed114230ecffe0a',
-				id: '6950565552966532158',
-			};
-
+			const recipientId = '13356260975429434553L';
+			const type = 0;
+			const amount = '10';
+			const fee = '10000000';
+			const senderPublicKey = '215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca';
+			const senderSecondPublicKey = '922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa';
```

Comment:
Can't we reuse some of the variables in the outer block?

Reply:
This is possible but not related to this PR and every signature for these tests would have to be redone.